app/models/event.rb

```diff
@@ -7,4 +9,9 @@ class Event < ApplicationRecord
   validates :capacity, presence: true, numericality: { greater_than: 0 }
 
   scope :active, -> { where cancelled: false }
+  scope :upcoming, -> { where 'time > ?', DateTime.now }
+
+  private def destroy_registrations
+    registrations.destroy_all
```

Comment:
Should we really delete registrations? We just cancelled the event, we should inform the participants.

Reply:
I have asked @IshmeetSingh06 to skip notifications for now, wanted project to be completed in this week.
Please let us know if we should integrate ethereal or just create dummy method for notfication?
@IshmeetSingh06 I think deleting the registrations is not the correct approach here, my bad if I told you to do so, ideally we should never delete the data.
Please remove registration removal code